src/main/scala/is/hail/annotations/OrderedRVIterator.scala

```diff
@@ -5,6 +5,15 @@ import is.hail.utils._
 
 case class OrderedRVIterator(t: OrderedRVDType, iterator: Iterator[RegionValue]) {
 
+  def restrictToPKInterval(interval: Interval): Iterator[RegionValue] = {
+    val ur = new UnsafeRow(t.rowType, null, 0)
+    val pk = new KeyedRow(ur, t.kRowFieldIdx)
+    iterator.filter( rv => {
```

Comment:
prefer curly braces around the entire thing instead of parens and curly braces together